scripts/src/main/resources/scripts/functions

```diff
@@ -423,10 +423,10 @@ function doInstall() {
   fi
   echo "Starting installation from ${2} to ${1}"
   doDownload "${2}"
-  doExtract ~/Downloads/${2/*\//} "${4}"
+  doExtract ~/Downloads/"${2/*\//}" "${4}"
   if [ -n "${3}" ]
   then
-    doMoveGlobSafe ${DEVON_IDE_HOME}/updates/extracted/${dir} ${DEVON_IDE_HOME}/updates/extracted/${3}
+    doMoveGlobSafe "${DEVON_IDE_HOME}/updates/extracted/${dir}" "${DEVON_IDE_HOME}/updates/extracted/${3}"
```

Comment:
This is wrong. ${3} may be a glob pattern.

Reply:
Ah - so I will move the end quote before the ${3}

Reply:
yep